src/py42/services/storage/exfiltrateddata.py

```diff
@@ -0,0 +1,49 @@
+from py42.services import BaseService
+
+
+class ExfiltratedDataService(BaseService):
+
+    _base_uri = u"api/v1/"
+
+    def __init__(self, main_session, streaming_session):
+        super(ExfiltratedDataService, self).__init__(main_session)
+        self._streaming_session = streaming_session
+
+    def get_download_token(self, event_id, device_id, file_path, timestamp):
+        """Get PDS download token for a file.
```

Comment:
Should this be `Get EDS download token for a file.`?

Reply:
yep